reana_job_controller/job_monitor.py

```diff
@@ -167,7 +167,7 @@ def get_job_status(self, job_pod):
         backend_job_id = self.get_backend_job_id(job_pod)
         if job_pod.status.phase == "Succeeded":
             logging.info("Kubernetes_job_id: {} succeeded.".format(backend_job_id))
-            status = "succeeded"
+            status = "finished"
```

Comment:
Related to https://github.com/reanahub/reana-workflow-engine-serial/pull/133#discussion_r580201633, not for this task.

Reply:
It makes total sense... I didn't want to start touching this yet because it opens a little Pandora's box as mentioned in the referenced comment. I would be more comfortable doing this as a separate PR group with more thorough testing after the workshop.

Reply:
Definitely, it was just to keep it in mind and maybe create an issue for it.

Reply:
Just saw the ticket 🤦 , nevermind.